source/cli.ts

```diff
@@ -45,13 +49,14 @@ const cli = meow(`
 		const cwd = cli.input.length > 0 ? cli.input[0] : process.cwd();
 		const typingsFile = cli.flags.typings;
 		const testFiles = cli.flags.files;
+		const {showDiff} = cli.flags;
```

Comment:
Coming back to this, realized I missed that you can rename during destructing:

```suggestion
		const {typings: typingsFile, files: testFiles, showDiff} = cli.flags;
```